### PR TITLE
Set task description to the overview screen

### PR DESCRIPTION
**Pull request by operando**
Set task description to the overview screen.

I used GooglePlay as reference.

## Reference Links

* http://developer.android.com/intl/ja/reference/android/app/Activity.html#setTaskDescription(android.app.ActivityManager.TaskDescription)
* http://qiita.com/pside/items/e28dfdd1f42f7a2cb9f2

out

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

**Comment by hotchemi** [comment]
LGTM! I didn't know that.

File: app/src/main/java/io/github/droidkaigi/confsched/util/AppUtil.java [hunk 1]
```diff
@@ -147,4 +150,16 @@ public static void showWebPage(Activity activity, @NonNull String url) {
         intent.launchUrl(activity, Uri.parse(url));
     }
 
+    public static void setTaskDescription(Activity activity, String label, int color) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            activity.setTaskDescription(new ActivityManager.TaskDescription(label, null, color));
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/util/AppUtil.java]: Wow, I didn't know this! Great.

**Comment by konifar** [comment]
Awesome LGTM!

**Comment by operando** [comment]
Thanks!!

**Pull request merged**